lib/matplotlib/backends/backend_pdf.py

```diff
@@ -1182,7 +1182,7 @@ def writeHatches(self):
                             0, 0, sidelen, sidelen, Op.rectangle,
                             Op.fill)
 
-            self.output(0.1, Op.setlinewidth)
+            self.output(rcParams['hatch.linewidth'], Op.setlinewidth)
```

Comment:
Fair point on being consistent, but there is currently no other API to change this value.  The main reason for getting this PR in for 2.0 is to put as rcparam in so we can change it and provide a way to get back to the old version.  Adding a proper API for changing this can be done later as a new feature and at that point it should be set at artist creation time, not draw time.